torch/distributed/distributed_c10d.py

```diff
@@ -816,7 +870,7 @@ def init_process_group(
             )
 
         default_pg = _new_process_group_helper(
-            -1, -1, [], Backend.MPI, None, group_name=group_name, timeout=timeout
+            -1, -1, [], backend, None, backend_config=backend_config, group_name=group_name, timeout=timeout
```

Comment:
Try use one of `backend` and `backend_config` only. Latter seems better.

Reply:
updated